packages/grid/x-data-grid/src/models/gridFilterItem.ts

```diff
@@ -28,4 +28,9 @@ enum GridLinkOperator {
   Or = 'or',
 }
 
-export { GridLinkOperator };
+enum GridQuickFilterLogic {
```

Comment:
Do we need two distinct enums ?

How about naming `quickFilterLogic` => `quickFilterLinkOperator` and keep a single enum ?

Reply:
I though two enums would be better, because it allows to add logic mode distinct from the logic operator in the future if requested, but not sure we will have such request

Reply:
We can start small and reuse `GridLinkOperator` until we need to support a logic operator exclusively in the quick filter.